### DISCUSSION TITLE
Problems while exporting the 'megamenu' function in 'mega-menu.tsx’

### DISCUSSION THREAD

**Discussion by sagarmusabbir** [themesberg/flowbite-react#1204] [Q&A] [answered]
I am new to Flowbite-React and encountering issues while exporting the 'megamenu' function in 'mega-menu.tsx,' a file included with my Flowbite Pro purchase. Despite installing Flowbite-React successfully, an error occurs when importing the 'megamenu' function from the node modules, stating that the function is unavailable. I've ensured the proper installation of Flowbite-React, and all other pages and components are functioning correctly. Could there be an existing problem or have I made an error? Apologies for my limited knowledge of Flowbite-React; I haven't opened an issue due to this. I would greatly appreciate an explanation of this issue.

**Comment by tulup-conner** [comment]
Hey @sagarmusabbir, the React Blocks are still under development and the reason that's happening is because Mega Menu is a feature we haven't added to the default latest release of `flowbite-react` yet. It's still under development, and you can use it now with your Blocks by installing `flowbite-react@beta`. Thanks for your patience on that - I intend to launch Mega Menu in the next major `flowbite-react` release.

**Reply by sagarmusabbir** [reply, marked as answer]
Thanks for the response. I will try to use the beta version for now.
